source/user-manual/certificates.rst

```diff
@@ -37,30 +37,41 @@ To create the certificates, edit the ``config.yml`` file and replace the node na
 
     .. code-block:: yaml
 
-       nodes:
-         # Wazuh indexer nodes
-         indexer:
-           - name: node-1
-             ip: <indexer-node-ip>
-           # - name: node-2
-           #   ip: <indexer-node-ip>
-           # - name: node-3
-           #   ip: <indexer-node-ip>
-       
-         # Wazuh server nodes
-         # Use node_type only with more than one Wazuh manager
-         server:
-           - name: wazuh-1
-             ip: <wazuh-manager-ip>
-           # node_type: master
-           # - name: wazuh-2
-           #   ip: <wazuh-manager-ip>
-           # node_type: worker
-       
-         # Wazuh dashboard nodes
-         dashboard:
-           - name: dashboard
-             ip: <dashboard-node-ip>
+         nodes:
+           # Wazuh indexer nodes
+           indexer:
+             - name: node-1
+               ip: <indexer-node-ip>
+             #- name: node-2
+             #  ip: <indexer-node-ip>
+             #- name: node-3
+             #  ip: <indexer-node-ip>
+
+           # Wazuh server nodes
+           # If there is more than one Wazuh server 
+           # node each one must have a node_type
+           server:
+             - name: wazuh-1
+               ip: <wazuh-manager-ip>
+             #  node_type: master
+             #- name: wazuh-2
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+             #- name: wazuh-3
+             #  ip: <wazuh-manager-ip>
+             #  node_type: worker
+
+
+          # Wazuh dashboard nodes
+          dashboard:
+            - name: dashboard
+              ip: <dashboard-node-ip>
+
+
```

Comment:
Idem

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/5727/commits/18d480774da4326aace3ae2b311a1e4805382607